### DISCUSSION TITLE
traefik starting fails

### DISCUSSION THREAD

**Discussion by zizo9999** [papermerge/papermerge-core#26] [Q&A]
Hi,
I have followed this guide:
https://docs.papermerge.io/Installation/portainer.html

all containers of the stack are starting. only traefik says"Request failed with status code 500" What could I do?

**Comment by ciur** [comment]
Try to change the ``APP_TAG`` and ``PAPERMERGE_JS_TAG`` to ``latest``. Could you also, please, paste logs here?

**Comment by zizo9999** [comment]
Hi,
I used latest for these TAGs.
I think I come closer to the problem.
I am running this on a synology nas and I think port 80 is used for something else.
If I change the traefik port to 81 it starts.
But papermerge.local stays on a white screen.
Do I have to config the 81 somewhere else?

IP:81 = „404 page not found“
IP:8080 = I see the traefik dashboard etc

I looked threw the logs:
One thing doesn't look ok:

papermerge-ws_server-1


      
```
93 static files copied to '/app/static'.
Operations to perform:
  Apply all migrations: auth, authtoken, contenttypes, core, dynamic_preferences, dynamic_preferences_users, knox, sessions, sites
Running migrations:
  No migrations to apply.
  Your models in app(s): 'core' have changes that are not yet reflected in a migration, and so won't be applied.
  Run 'manage.py makemigrations' to make new migrations, and then re-run 'manage.py migrate' to apply them.
Creating superuser username=adminzob
CommandError: Error: That username is already taken.
INFO 2022-07-14 18:54:36,092 cli Starting server at tcp:port=8000:interface=0.0.0.0
INFO 2022-07-14 18:54:36,096 server HTTP/2 support not enabled (install the http2 and tls Twisted extras)
INFO 2022-07-14 18:54:36,097 server Configuring endpoint tcp:port=8000:interface=0.0.0.0
INFO 2022-07-14 18:54:36,100 server Listening on TCP address 0.0.0.0:8000

```
